spec/requests/api/v1/test_ecf_participants_spec.rb

```diff
@@ -135,7 +135,7 @@
           end
         end
 
-        context "multiple profiles" do
+        xcontext "multiple profiles", "Fix in later PR: the new query does not account yet for multiple profiles although this should not happen" do
```

Comment:
it should be noted that marking this as pending will cause providers to see multiple records for users that have more than one profile. it ties in with my other comment, https://github.com/DFE-Digital/early-careers-framework/pull/2349/files#r936818079